samples/SimpleDataSource/SimpleCustomDataSource.cs

```diff
@@ -41,6 +41,65 @@ public class SimpleCustomDataSource
     {
         private IApplicationEnvironment applicationEnvironment;
 
+        //
+        // Provides information about this Custom Data Source, such as the author,
+        // a project link, licensing, etc. This information can be used by tools
+        // to display "About" information for your Custom Data Source. For example,
+        // Windows Performance Analyzer (WPA) uses this information for Help->About.
+        //
```

Comment:
Should we include this comment in the SqlCustomDataSource file as well?

Reply:
No, as the other sample don't have all of the other comments either, it would be out of place.